src/rust/madara/sources/galaxydegenscans/res/source.json

```
@@ -0,0 +1,18 @@
+{
+	"info": {
+		"id": "en.galaxydegenscans",
+		"lang": "en",
+		"name": "Galaxy Degen Scans",
+		"version": 1,
+		"url": "https://gdscans.com/",
```

shouldn't have a trailing slash, otherwise lgtm
```suggestion
		"url": "https://gdscans.com",
```